src/main/java/uk/ac/ebi/eva/contigalias/entities/ChromosomeId.java

```diff
@@ -0,0 +1,8 @@
+package uk.ac.ebi.eva.contigalias.entities;
+
+import java.io.Serializable;
+
+public class ChromosomeId implements Serializable {
+    private String insdcAccession;
+    private String assembly;
```

Comment:
Just for my understanding: this is type `String` because while an `AssemblyEntity` was annotated `@Id` in the main entity class, it's really using the id of `AssemblyEntity` which is `String insdcAccession`.... is that right?